core/src/main/java/com/datastax/oss/driver/api/core/tracker/RequestTracker.java

```diff
@@ -51,7 +78,7 @@ default void onSuccess(
       long latencyNanos,
       @NonNull DriverExecutionProfile executionProfile,
       @NonNull Node node,
-      @NonNull ExecutionInfo executionInfo,
+      @Nullable ExecutionInfo executionInfo,
```

Comment:
Wondering why this changed from `@NonNull` to `@Nullable`? Was it previously improperly annotated or was there a change the implementation that now makes it nullable?

Reply:
Initially `CqlPrepareHandler` did not generate `ExecutionInfo`, but I have added support for it now. Fixed.